test-fixtures/all/variables.tf

```diff
@@ -24,3 +24,12 @@ variable "local_ip_addresses" {
   description = "The external IP addresses of the machines running the acceptance tests. This is necessary to allow access to the Azure SQL Server resource."
   type        = list(string)
 }
+
```

Comment:
This is odd, you need this variable only to run the local tests, where the sql server is started in the Docker container. Is there no better fix for this?